library/src/main/scala-new-collections/scala/scalajs/js/ArrayOps.scala

```diff
@@ -889,8 +889,10 @@ final class ArrayOps[A](private val xs: js.Array[A]) extends AnyVal {
     while (i < len) {
       matched = true
       val v = f.applyOrElse(xs(i), d)
+      // scalastyle:off return
```

Comment:
Those should be at the very beginning and very end of the `def`. In fact they're already there, but misspelled. Please fix those instead of introducing new ones.